doc/de/Account-Basics.md

```diff
@@ -109,7 +109,7 @@ Ein ['Tipp für neue Mitglieder'](newmember)-Link zeigt sich in den ersten beide
 
 ## Schau Dir ebenfalls folgende Seiten an
 
-* [Gruppen und Privatssphäre](help/Groups-and-Privacy)
+* [Circles und Privatssphäre](help/Circles-and-Privacy)
```

Comment:
I'm not comfortable making this change on my own, a few German people said "Circle" was fine.

Reply:
I would prefer Circles too, Kreise seems too much direct translated into German.

Reply:
I'm surprised that "Circles" is the preferred choice. Although I have near-native speaking ability, I wouldn't exactly know what German people would say.

Either way, I'm puzzled about the plural of "Circles" in German. Does it not follow the logic of "Kreise" (singular) -  "Kreise" (plural)? Is that how people really talk?

Considering the fact that Nextcloud uses the German term, I'd say there should be a discussion  (with a consensus or vote based outcome) in the German-speaking community on the exact translation.

Reply:
Don't we use the word Freundeskreis anymore?

Reply:
@AndyHee There was a discussion, and you contributed to it: https://github.com/friendica/friendica/issues/13114#issuecomment-1546939430

Reply:
This might really be the place here to have that discussion. Possibly better to talk about it in the [Support forum](https://forum.friendi.ca/profile/helpers) in German.

But on a sidenote, I once heard an older German gentleman saying: "_In welchen Kreisen verkehren Sie eigentich?_" Which means something like 'In what circles are you socialising?'. Apparently it's a polite way of expressing disapproval. 😲

Reply:
@MrPetovan Sorry I mean a discussion about the correct German translation. 

Are you worried that it will be hijacked and turned into a discussion of whether to change the terminology or not? 

(I'm only seeing your reply now. Why?)

Reply:
Maybe for practical purposes, let's just push this through and revisit the translation issue later?

As said, I'm surprised about the choice of translation. But I'm not on that translation team and my knowledge is limited.

How to proceed? @MrPetovan @nupplaphil @Quix0r?

Reply:
I'm disappointed you are surprised because your participated in the issue discussion where we talked about it and I received two separate confirmations by German people in the same conversation that "Circle" was fine for the German documentation.

The worst that could happen right now is if this PR is blocked by yet another language debate after I sought and received an answer and put work to it.

Reply:
Yes, I noticed that some German people wanted to settle on "Circle". Today I saw what this really looks like. Hence, I added my comment.

I think you're right. It's not wise to mix these discussions. I understand that you moved the German documentation into this PR for swiftly moving this forward. But it might have contributed to the little difficulty. 

So, let push this through, and we'll leave it to the German translation team to review it.

I'm sorry that you felt disappointed.

Reply:
The issue here is that the German documentation pages aren't part of the regular translation workflow, meaning that developers have to make these decisions instead of the translators over at Transifex. I'm not sure if there's a way to include the documentation pages in Transifex, nor if it's desirable.

Reply:
Sure, I probably would have done exactly the same. Let's not worry too much about this.